python/sdssdb/sqlalchemy/sdss5db/opsdb.py

```diff
@@ -0,0 +1,265 @@
+# !usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+
+# this file was autogenerated with sqlacodegen and then adapted for sdssdb
+
+import os
+
+from sqlalchemy import (ARRAY, BigInteger, Boolean, Column, Date, DateTime, Float,
+                        ForeignKey, Integer, SmallInteger, Text, text)
+from sqlalchemy.orm import relationship
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.ext.declarative import AbstractConcreteBase, declared_attr
+
+from sdssdb.sqlalchemy.sdss5db import SDSS5dbBase, database
+
+
+class Base(AbstractConcreteBase, SDSS5dbBase):
+    __abstract__ = True
+    _schema = 'opsdb_apo' if os.getenv("OBSERVATORY") == 'APO' else 'opsdb_lco'
+    _relations = 'define_relations'
+
+    @declared_attr
+    def __table_args__(cls):
+        return {'schema': cls._schema}
+
+
+class BasePriority(Base):
+    __tablename__ = 'base_priority'
+
+    pk = Column(Integer, primary_key=True, server_default=text(f"nextval('{Base._schema}.base_priority_pk_seq'::regclass)"))
+    field_pk = Column(Integer)
+    priority = Column(Integer)
+    version_pk = Column(Integer)
+
+
+class CompletionStatus(Base):
+    __tablename__ = 'completion_status'
+
+    pk = Column(Integer, primary_key=True, server_default=text(f"nextval('{Base._schema}.completion_status_pk_seq'::regclass)"))
+    label = Column(Text)
+
+
+class ExposureFlavor(Base):
+    __tablename__ = 'exposure_flavor'
+
+    pk = Column(Integer, primary_key=True, server_default=text(f"nextval('{Base._schema}.exposure_flavor_pk_seq'::regclass)"))
+    label = Column(Text)
+
+
+class FieldPriority(Base):
+    __tablename__ = 'field_priority'
+
+    pk = Column(Integer, primary_key=True, server_default=text(f"nextval('{Base._schema}.field_priority_pk_seq'::regclass)"))
+    label = Column(Text)
+
+
+class PriorityVersion(Base):
+    __tablename__ = 'priority_version'
+
+    pk = Column(Integer, primary_key=True, server_default=text(f"nextval('{Base._schema}.priority_version_pk_seq'::regclass)"))
+    label = Column(Text)
+
+
+class Survey(Base):
+    __tablename__ = 'survey'
+
+    pk = Column(Integer, primary_key=True, server_default=text(f"nextval('{Base._schema}.survey_pk_seq'::regclass)"))
+    label = Column(Text)
+
+
+class DesignMode(Base):
```

Comment:
I don't know why this was included in opsdb, it should be in targetdb and I don't see it in opsdb_apo at utah?